spec/aerospike/command_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Castle, Inc.
```

Comment:
Will this work if we want to merge this upstream?

Reply:
Just tell me what you want here and I will adjust it accordingly.

Reply:
@ioquatix just remove it :) As we agreed during the contract signing, this is going to be merged to the upstream and OSS, so no need for this comment. We can always squash and amend the commit message to indicate that it was done by you on our behalf.

Reply:
The upstream code has copyright assignment on a per-file basis: https://github.com/aerospike/aerospike-client-ruby/blob/master/lib/aerospike/key.rb#L2-L15

So, I just tried to replicate that. I can put my own name or remove it entirely but to remove it would be different from upstream conventions. Let me know what you'd prefer.

Reply:
I've left the license but removed the Copyright which can be sorted out later.